deploy.yaml

```diff
@@ -0,0 +1,27 @@
+#To use this you will need buildcomponents.yaml in your top level ci file. 
```

Comment:
Can you name the file more-specifically than "deploy". Either something with the word "elastic beanstalk" or "grunt commerce deploy"?

Because I want another one for serverless_deploy